mbuild/compound.py

```diff
@@ -126,6 +126,14 @@ def load(filename_or_object, relative_to_module=None, compound=None, coords_only
             compound = read_xyz(filename_or_object, compound=compound)
         return compound
 
+    if extension == '.sdf':
+        pybel = import_('pybel')
+        pybel_mol = pybel.readfile('sdf', filename_or_object)
+        # Currently only reading in single molecules
+        single_mol = [i for i in pybel_mol][0]
```

Comment:
Let's raise an exception if multiple molecules are found. It looks like that object is iterable so maybe `if len(pybel) > 1`?

Reply:
is generic `raise Exception` okay here?

Reply:
I think this fits the definition of `ValueError`?

Reply:
I'm also not sure about warning vs. erroring out - my first thought was to error out but I could see it either way.

Reply:
I misunderstood the way pybel returns generators.  It seems like pybel returns a single `molecule` object even for files containing multiple molecules.  I added a test to ensure this behavior.

Right now, I'm handling this similarly to the smiles implementation.  That is, if the len(pybel_generator) is greater than 1, the user will be warned of this.